resources/openapi/yaml/management-api/contract-definition-api.yaml

```diff
@@ -354,7 +354,25 @@ paths:
         content:
           '*/*':
             schema:
-              $ref: '#/components/schemas/QuerySpecDto'
+              type: object
```

Comment:
in the `Api` interface the request body could be set as:
`requestBody = @RequestBody(content = @Content(schema = @Schema(implementation = QuerySpecDto.class)))`

it won't document the whole json-ld thing, but it will better than the `JsonValue` exploded view